### PR TITLE
UG final v1.4

### LINKED ISSUE #169: [PE-D] UG: cadd - Expected Output not specified
1. Expected output from example commands not specified for reader
2. Screenshot:

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: alinaleehx/ped#2

### LINKED ISSUE #170: [PE-D] UG: cfind - example output not accurate
In the UG, it is written that searches for "e/gmail" will return contacts with emails that contain "gmail.com":

However, a search with "e/gmail" also returns contacts with "gmail.[other text]", and not just "gmail.com"

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: mweeruien/ped#1

### LINKED ISSUE #171: [PE-D] UG: eadd - Incorrect example for end time
When I try pasting this command into the cli and upon pressing enter, I get the following error:

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: wilburrito/ped#1

### LINKED ISSUE #179: [PE-D] UG: Events without end date and time behaviour not specified
Overflows to the next day

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: Bluntsord/ped#5

### LINKED ISSUE #187: [PE-D] UG: Confusing examples
In some of the examples, I found myself a bit lost especially when you are talking about using 2 features one after the other. Perhaps add more visuals so the user can follow.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: luluyousef/ped#3

### LINKED ISSUE #188: [PE-D] UG: Copying Links Bug
In the UG, it is stated that any details can be copied by clicking on it.

However, clicking on the telegram handle of "Bernice Yu" does not copy the telegram handle, but instead opens it as a telegram link in the browser.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: mweeruien/ped#4

### LINKED ISSUE #191: [PE-D] UG: Capitalization in exit command
Could make 'Close' not capitalized for consistency

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: luluyousef/ped#4

### LINKED ISSUE #192: [PE-D] UG: Lack of screenshots in user guide
1. Insufficient screenshots of expected GUI. For example for all commands in contact management only 1 had screenshot. This will make it hard for readers to know what the correct expected output is.

Example:

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: alinaleehx/ped#6

### LINKED ISSUE #193: [PE-D] UG: Grammatical error in description of Undo
instead of 'Restore' -> 'Restores'

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: luluyousef/ped#5

### LINKED ISSUE #195: [PE-D] UG: Glossary has nothing in there
Suggestion: Should add and explain some words here

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: alinaleehx/ped#7

### LINKED ISSUE #196: [PE-D] UG: Numbered listing in Instructions
It might look neater and easier to follow if this was in a numbered list.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: luluyousef/ped#6

### LINKED ISSUE #198: [PE-D] Grammatical error
-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: luluyousef/ped#7

### LINKED ISSUE #199: [PE-D] UG: Uniqueness of Contacts when doing cadd
Not sure how you establish the uniqueness of your contact objects (i assume it's smth like name.equals(otherName) && email.equals(otherEmail))

But consider this: Say I have an Alex Doe in my contacts already, but I want to add another person who coincidentally has the exact same name, but his email is different (because he's a different person, but unfortunately he really has the same name)

Your cadd command tells me that the second Alex Doe already exists in my contacts, which isn't exactly true I suppose. 

So maybe some way to handle this kind of uniqueness can be addressed either as a bug fix, or maybe just address how to handle 2 people with the same name in the user guide, under the cadd command.

-------------
Labels: `severity.VeryLow` `type.FunctionalityBug`
original: wilburrito/ped#7

### PR DESCRIPTION
**Pull request by chunweii**
- [x] Add tutorial on reading commands
- [x] Add redo
- [x] closes #169 
- [x] closes #170 
- [x] closes #171
- [x] closes #179 
- [x] closes #187 
- [x] closes #188 
- [x] closes #191
- [x] closes #192 
- [x] closes #193 
- [x] closes #195 
- [x] closes #196
- [x] closes #198 
- [x] closes #199

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
+`cunmark 1 2` | Unmarks the first and second contact of the **currently displayed** contact list in SoConnect. <br><br> You should see this message in the message box:<br> `Unmarked contact: Irfan Ibrahim; Email: irfan@example.com; Address: Blk 47 Tampines Street 20, #17-35; Telegram: irfanx; Tags: [classmates] Unmarked contact: David Li; Email: lidavid@comp.nus.edu.sg; Address: COM1-B1-0931; Telegram: lidavid; Tags: [professor][CS2103T]` <br><br> You should see `Irfan Ibrahim` and `David Li` moved in that order to the **bottom of your marked contact list** without the **Bookmark Icon**: ![Unmark Contact](images/demo-screenshots/cunmarkEx2.png)
```
Comment (Gordon25) [inline docs/UserGuide.md:501]: cunmarkEx2

The screenshot should be this updated one.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
 Adds an event to SoConnect.
 
 **Format:** `eadd n/NAME at/START_TIME [end/END_TIME] [d/DESCRIPTION] [a/ADDRESS] [z/ZOOM] [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-An event can have any number of tags (including 0)
+<div markdown="block" class="alert alert-primary">:bulb: **Tip:**
+
+* An event can have any number of tags (including 0)
+* You **cannot** add an event with the same name as an existing event.
```
Comment (Gordon25) [inline docs/UserGuide.md:557]: Can bold same name

File: docs/UserGuide.md [hunk 3]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
 </div>
 
 
-### Deleting an event: `edelete`
+#### Deleting an event: `edelete`
 
-Deletes the specified event from SoConnect.
+Deletes the specified event(s) from SoConnect.
 
 **Format:** `edelete INDEX1[-INDEX2]`
```
Comment (Gordon25) [inline docs/UserGuide.md:590]: Should change to `INDEX [-INDEX]`

File: docs/UserGuide.md [hunk 4]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
 
-* Deletes the contact at:
+* Deletes the event(s) at:
     * the specified `INDEX1` or
     * between the specified range from `INDEX1` to `INDEX2` inclusively (if you provide `INDEX2`).
-* `INDEX` refers to the index number shown in the displayed event list.
+* `INDEX1` and `INDEX2` refer to the index numbers shown in the displayed event list.
```
Comment (Gordon25) [inline docs/UserGuide.md:595]: the first and second `INDEX`es refer to

File: docs/UserGuide.md [hunk 5]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
+
+You must provide **at least one** of the optional _fields_.
+</div>
+
 * Edits the event at the specified `INDEX`.
 * `INDEX` refers to the index number shown in the displayed event list.
-* You must provide **at least one** of the optional fields.
 * Existing values will be updated to the input values.
 * You can use `t/` to add a tag.
 * You can remove a specific tag by typing `dt/` followed by the tag name that you wish to remove.
```
Comment (Gordon25) [inline docs/UserGuide.md]: same as contact, "the name of the tag you wish to remove"

File: docs/UserGuide.md [hunk 6]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
 
-**Format:** `cdelete INDEX1[-INDEX2]`
+**Format:** `cdelete INDEX[-INDEX]`
 
-* Deletes the contact at:
-  * the specified `INDEX1` or 
-  * between the specified range from `INDEX1` to `INDEX2` inclusively (if you provide `INDEX2`).
+* Deletes the contact(s):
+    * at the specified `INDEX` or
+    * between the specified range from the first `INDEX` to the second `INDEX2` inclusively (if you provide the second `INDEX`).
```
[the remark this answers is not in the document]
Comment (xiangjunn) [reply docs/UserGuide.md]: I changed to `INDEX1` and `INDEX2` to follow the application message usage.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -19,472 +23,664 @@ full range of features it offers.
...
+#### Deleting an event: `edelete`
 
-Deletes the specified event from SoConnect.
+Deletes the specified event(s) from SoConnect.
 
 **Format:** `edelete INDEX1[-INDEX2]`
 
-* Deletes the contact at:
+* Deletes the event(s) at:
     * the specified `INDEX1` or
```
[the remark this answers is not in the document]
Comment (xiangjunn) [reply docs/UserGuide.md:593]: This is correct.

File: docs/UserGuide.md [hunk 8]
```diff
@@ -633,7 +633,7 @@ You must provide **at least one** of the optional _fields_.
 * `INDEX` **must be a positive integer**. e.g. 1, 2, 3, …
 * Start time and End Time should be of format **dd-MM-yyyy HH:mm** (date-MONTH-year HOUR:minute in 24-hour format).
 * End Time should be **chronologically after** the Start Time.
-* You **cannot** edit an event to the same name as an existing event.
+* You **cannot** edit an event to the **same name** as an existing event.
 </div>
 
 **Examples:**
```
Comment (Gordon25) [inline docs/UserGuide.md:639]: for eunmark second example, can include the link back to cunmark also

**Review by pcgiang — APPROVED** [review]

**Review by Gordon25 — APPROVED** [review]

**Pull request merged**